frontend/www/js/omegaup/components/user/Edit.vue

```diff
@@ -0,0 +1,408 @@
+            <h3 class="card-title">{{ T.userEditEditProfile }}</h3>
+          </div>
+          <form role="form" class="card-body" @submit.prevent="onUpdateUser">
+            <div class="form-group">
+              <label>{{ T.username }}</label>
+              <input
+                v-model="selectedProfileInfo.username"
+                class="form-control"
+              />
+            </div>
+            <div class="form-group">
+              <label>{{ T.wordsName }}</label>
+              <input v-model="selectedProfileInfo.name" class="form-control" />
+            </div>
+            <div class="form-group">
+              <label>{{ T.userEditBirthDate }}</label>
+              <font-awesome-icon
+                :title="T.courseNewFormStartDateDesc"
+                icon="info-circle"
+              />
+              <omegaup-datepicker
+                v-model="selectedProfileInfo.birth_date"
+              ></omegaup-datepicker>
+            </div>
+            <div class="form-group">
+              <label>{{ T.wordsGender }}</label>
+              <select
+                v-model="selectedProfileInfo.gender"
+                name="gender"
+                class="form-control"
+              >
+                <option value="female">{{ T.wordsGenderFemale }}</option>
+                <option value="male">{{ T.wordsGenderMale }}</option>
+                <option value="other">{{ T.wordsGenderOther }}</option>
+                <option value="decline">{{ T.wordsGenderDecline }}</option>
+              </select>
+            </div>
+            <div class="form-group">
+              <label>{{ T.wordsCountry }}</label>
+              <select
+                v-model="selectedProfileInfo.country_id"
+                name="country_id"
+                class="form-control"
+              >
+                <option
+                  v-for="country in countries"
+                  :key="country.country_id"
+                  :value="country.country_id"
+                >
+                  {{ country.name }}
+                </option>
+              </select>
+            </div>
+            <div class="form-group">
+              <label>
+                {{ T.profileState }}
+              </label>
+              <select
+                v-model="selectedProfileInfo.state_id"
+                class="form-control"
+              >
+                <option
+                  v-for="[code, state] in Object.entries(countryStates)"
+                  :key="code"
+                  :value="code.split('-')[1]"
+                >
+                  {{ state.name }}
+                </option>
+              </select>
+            </div>
+            <div class="form-group">
+              <label class="d-block">
+                {{ T.profileSchool }}
+                <omegaup-common-typeahead
+                  class="w-100"
+                  :existing-options="schools"
+                  :value.sync="participant"
```

Comment:
Por consistencia con otros componentes, en `existing-options` se puede cambiar `schools` por `searchResultSchools`?

Otra cosa, estás utilizando un objeto con todos los atributos del perfil, aquí también podrías usar en lugar de `participant`, este otro: `selectedProfileInfo.school`. Y tomando en cuenta que para guardar la escuela se requiere enviar a la API el `school_id`, puedes agregar  `id.sync`. Debería quedar algo como esto:
```suggestion
                  :existing-options="schools"
                  :value.sync="participant"
                  :id.sync="selectedProfileInfo.school_id"
```